client/src/components/tools/SortDropDown.tsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from "react"
+import { useSearchParams } from "react-router-dom";
+import SortDropDownMenu from "./SortDropDownMenu"
+
+type Params = {
+  order: string,
+  sort: string
+}
+
+const sortOptions: string[] = ["name", "author", "publication_date"];
+const byOptions: string[]  = ["asc","desc"]
```

Comment:
![image](https://user-images.githubusercontent.com/83677410/229033319-cdc581cb-b0c0-42cb-a8ab-63154a41d78b.png)


could we display these as **acceding** and **descending** in the dropdown?

```ts
const setOrder = (value: string) => {
  setParams({ ...params, order: value === 'ascending' ? 'asc' : 'desc' });
}
```

Reply:
Good call, I have added this